videostore/rawsegmenter.c

```diff
@@ -8,47 +8,36 @@
 #include <stdint.h>
 #include <stdio.h>
 #include <string.h>
-int video_store_raw_seg_init_h264(struct raw_seg_h264 **ppRS, // OUT
-                                  const int segmentSeconds,   // IN
-                                  const char *outputPattern,  // IN
-                                  const char *extradata,      // IN
-                                  const size_t extradataSize, // IN
-                                  const int width,            // IN
-                                  const int height            // IN
+int video_store_raw_seg_init(struct raw_seg **ppRS,     // OUT
```

Comment:
we no longer need exradata we now expect the callers of `video_store_raw_seg_write_packet` to send what was previously sent in extradata at the beginning of the packet.